ssl/s3_enc.c

```diff
@@ -589,6 +589,8 @@ int ssl3_alert_code(int code)
         return TLS1_AD_NO_APPLICATION_PROTOCOL;
     case SSL_AD_CERTIFICATE_REQUIRED:
         return SSL_AD_HANDSHAKE_FAILURE;
+    case TLS13_AD_MISSING_EXTENSION:
+        return TLS13_AD_MISSING_EXTENSION;
```

Comment:
Wouldn't it be better to map this to an actual SSLv3 alert like `SSL3_AD_HANDSHAKE_FAILURE`?

Reply:
At present, the only way to get this alert is if you sent an extension first defined for TLS 1.3 and only useful for TLS 1.3, so you could reasonably be assumed to know what to do with the TLS 1.3 alert ... but I also don't actually care about this scenario.